src/registrar/models/models.py

```diff
@@ -83,3 +87,179 @@ def __str__(self):
+    DOMAIN_REGEX = re.compile(r"^(?!-)[A-Za-z0-9-]{1,63}(?<!-)\.[A-Za-z]{2,6}")
+
+    @classmethod
+    def string_could_be_domain(cls, domain):
+        """Return True if the string could be a domain name, otherwise False.
+
+        TODO: when we have a Domain class, this could be a classmethod there.
+        """
+        if cls.DOMAIN_REGEX.match(domain):
+            return True
+        return False
+
+
+class Contact(models.Model):
+
+    """Contact information follows a similar pattern for each contact."""
+
+    first_name = models.TextField(null=True, help_text="First name")
+    middle_name = models.TextField(null=True, help_text="Middle name")
+    last_name = models.TextField(null=True, help_text="Last name")
+    title = models.TextField(null=True, help_text="Title")
+    email = models.TextField(null=True, help_text="Email")
+    phone = models.TextField(null=True, help_text="Phone")
+
+
+class DomainApplication(TimeStampedModel):
+
+    STARTED = "started"
+    SUBMITTED = "submitted"
+    INVESTIGATING = "investigating"
+    APPROVED = "approved"
+    STATUS_CHOICES = [
+        (STARTED, STARTED),
+        (SUBMITTED, SUBMITTED),
+        (INVESTIGATING, INVESTIGATING),
+        (APPROVED, APPROVED),
+    ]
+    status = FSMField(
+        choices=STATUS_CHOICES,  # possible states as an array of constants
+        default=STARTED,  # sensible default
+        protected=True,  # cannot change state directly, must use methods!
+    )
+    creator = models.ForeignKey(
+        User, on_delete=models.PROTECT, related_name="applications_created"
+    )
+    investigator = models.ForeignKey(
+        User,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="applications_investigating",
+    )
+
+    # data fields from the initial form
+
+    FEDERAL = "federal"
+    INTERSTATE = "interstate"
+    STATE_OR_TERRITORY = "state_or_territory"
+    TRIBAL = "tribal"
+    COUNTY = "county"
+    CITY = "city"
+    SPECIAL_DISTRICT = "special_district"
+    ORGANIZATION_CHOICES = [
+        (FEDERAL, "a federal agency"),
+        (INTERSTATE, "an organization of two or more states"),
+        (
+            STATE_OR_TERRITORY,
+            "one of the 50 U.S. states, the District of "
+            "Columbia, American Samoa, Guam, Northern Mariana Islands, "
+            "Puerto Rico, or the U.S. Virgin Islands",
+        ),
+        (
+            TRIBAL,
+            "a tribal government recognized by the federal or " "state government",
+        ),
+        (COUNTY, "a county, parish, or borough"),
+        (CITY, "a city, town, township, village, etc."),
+        (SPECIAL_DISTRICT, "an independent organization within a single state"),
+    ]
+    organization_type = models.CharField(
+        max_length=255, choices=ORGANIZATION_CHOICES, help_text="Type of Organization"
+    )
+
+    EXECUTIVE = "Executive"
+    JUDICIAL = "Judicial"
```

Comment:
**Suggestion:** I kinda dislike having constants mixed with field declarations. If you like it better this way, I'm fine with it, but if you don't care, let's put constants at the top.